fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -531,6 +529,17 @@ class PythonUtils {
         «setAttrPropertyHL("standard_unit", attr.properties.standardUnit, true)»
         «setAttrPropertyHL("display_unit", attr.properties.displayUnit, true)»
         «setAttrPropertyHL("format", attr.properties.format.formatComaToPoint, true)»
+        «setAttrPropertyHL("polling_period", attr.polledPeriod, false)»
```

Comment:
Will this only add the text like `polling_period=123` if the value is > 0?
What does the last parameter do?  I see it is false for some lines and true for others.